src/components/presentation/NavBar.test.tsx

```diff
@@ -123,7 +123,9 @@ describe('NavBar', () => {
     });
     await waitForElementToBeRemoved(screen.queryByRole('progressbar'));
     const dropdown = screen.getByRole('combobox', { name: /Språk/i });
-    await act(() => userEvent.selectOptions(dropdown, 'en'));
+    await act(() => userEvent.click(dropdown));
+    const en = screen.getByText(/Engelsk/i, { selector: '[role=option]' });
+    await act(() => userEvent.click(en));
```

Comment:
The act should not be necessary, because RTL has already wrapped those events within an act internally.
```suggestion
    await userEvent.click(en);
```